utils/misc.py

```diff
@@ -151,3 +151,30 @@ def download_ampere_imagenet():
 
     os.environ["IMAGENET_IMG_PATH"] = str(dataset)
     os.environ["IMAGENET_LABELS_PATH"] = str(labels)
+
+
+def download_coco_dataset():
+    from utils.downloads.utils import get_downloads_path
+    if "COCO_IMG_PATH" not in os.environ and "COCO_ANNO_PATH" not in os.environ:
```

Comment:
This is not a good test because user has the alternative ability to supply COCO paths through args instead of env variables. The execution of this function should be decided by COCODataset having ultimate knowledge of whether user has supplied paths on his own or not.

Reply:
This will also save the effort of putting this function's call into each and every COCO related run.py script.